pkgs/checks/lib/src/checks.dart

```diff
@@ -433,7 +433,7 @@ abstract final class Context<T> {
   ///   }
   /// }
   /// ```
-  Future<void> expectAsync<R>(Iterable<String> Function() clause,
```

Comment:
@natebosch – `<R>` seems superfluous! See overrides below, too!

Reply:
Huh, I think this is a leftover from a _very_ early design where `nest` didn't exist. I'm surprised it survived so long without getting noticed.